package NAlice.NWonderlogs;

import "alice/library/field_differ/protos/differ_report.proto";

import "mapreduce/yt/interface/protos/extension.proto";

message TWonderlogsDiff{
    option (NYT.default_field_flags) = SERIALIZATION_YT;
    reserved 4, 5;
    reserved "Stable", "Test";

    optional string Uuid = 1 [json_name = "uuid", (NYT.column_name) = "uuid"];
    optional string MessageId = 2 [json_name = "message_id", (NYT.column_name) = "message_id"];
    optional NAlice.TDifferReport Diff = 3 [json_name = "diff", (NYT.column_name) = "diff"];
    optional string StableWonderlog = 7 [json_name = "stable_wonderlog", (NYT.column_name) = "stable_wonderlog"];
    optional string TestWonderlog = 8 [json_name = "test_wonderlog", (NYT.column_name) = "test_wonderlog"];
    optional string MegamindRequestId = 9 [json_name = "megamind_request_id", (NYT.column_name) = "megamind_request_id"];
}
